src/screens/SendFunds/04-Summary.js

```diff
@@ -161,11 +166,27 @@ function SendSummary({ navigation, route: initialRoute }: Props) {
   const { transaction: transactionError } = errors;
   const error = status.errors[Object.keys(status.errors)[0]];
   const mainAccount = getMainAccount(account, parentAccount);
+  const currency = getAccountCurrency(account);
+  const hasNonEmptySubAccounts =
+    account.type === "Account" &&
+    some(account.subAccounts, subAccount => subAccount.balance.gt(0));
```

Comment:
- I would prefer we use JavaScript `array.some`
- subAccounts could be undefined here.